packages/shorebird_cli/lib/src/commands/preview_command.dart

```diff
@@ -206,7 +209,42 @@ class PreviewCommand extends ShorebirdCommand
   }
 
   Future<int> installAndLaunchIos(String appId, Release release) async {
-    return ExitCode.unavailable.code;
+    const platform = ReleasePlatform.ios;
+    final previewDirectory = cache.getPreviewDirectory(appId);
+    final runnerPath = p.join(
+      previewDirectory.path,
+      '${platform.name}_${release.version}.app',
+    );
+
+    if (!Directory(runnerPath).existsSync()) {
+      final downloadArtifactProgress = logger.progress('Downloading release');
+      try {
+        final releaseRunnerArtifact =
+            await codePushClientWrapper.getReleaseArtifact(
+          appId: appId,
+          releaseId: release.id,
+          arch: 'runner',
```

Comment:
Did you mean arch runner?

Reply:
You'll want a named constant at least.

Reply:
Yeah this is the naming convention. I think we can create some constants to clean this up in a subsequent PR.